core/tests/protractor_utils/StoryEditorPage.js

```diff
@@ -155,7 +155,7 @@ var StoryEditorPage = function() {
   };
 
   this.unpublishStory = async function() {
-    await unpublishStoryButton.click();
+    await unpublishStoryButton.action.click();
```

Comment:
I think what you want to write here is: `await action.click('Unpublish Story Button', unpublishStoryButton)`

Reply:
Done.